specs/wp-signup-spec.js

```diff
@@ -271,96 +229,86 @@ testDescribe( `[${host}] Sign Up  (${screenSize}, ${locale})`, function() {
 		} );
 
 		test.it( 'We can set the sandbox cookie for payments', function() {
-			this.WPHomePage = new WPHomePage( driver, { visit: true, culture: locale } );
+			const wPHomePage = new WPHomePage( driver, { visit: true, culture: locale } );
 			eyesHelper.eyesScreenshot( driver, eyes, 'Logged Out Homepage' );
-			return this.WPHomePage.setSandboxModeForPayments( sandboxCookieValue );
+			return wPHomePage.setSandboxModeForPayments( sandboxCookieValue );
 		} );
 
 		test.describe( `Step ${stepNum}: About Page`, function() {
 			stepNum++;
 
 			test.it( 'Can see the about page', function() {
-				this.startPage = new StartPage( driver, { visit: true, culture: locale } );
-				this.aboutPage = new AboutPage( driver );
-				return this.aboutPage.displayed().then( ( displayed ) => {
+				new StartPage( driver, { visit: true, culture: locale } ).displayed();
```

Comment:
Should we also break this out into its own step, as in the previous test (lines 87-95)?

Reply:
Good idea
I didn't enter any of the "about" information in this scenario, I thought entering a random site title could cause the visual difference tests to complain about the dynamic information? What do you think? Just submit that form as we did before?

Reply:
Yes, agreed, I wouldn't enter any "About" information here. If we wanted to enter information I'd do it after the screenshot is taken for the visual diff test, but also I think it's worth having a test that verifies you can continue without entering information here.